e2e-tests/trade-finance-manager/cypress/integration/journeys/case/deal-MIA.spec.js

```diff
@@ -0,0 +1,81 @@
+import relative from '../../relativeURL';
+import pages from '../../pages';
+import partials from '../../partials';
+import MOCK_DEAL_MIA from '../../../fixtures/deal-MIA';
+import MOCK_USERS from '../../../fixtures/users';
+import { MOCK_MAKER_TFM, ADMIN_LOGIN } from '../../../fixtures/users-portal';
+
+context('User can view a case deal', () => {
+  let dealId;
+  let dealFacilities = [];
+
+  before(() => {
+    cy.deleteDeals(MOCK_DEAL_MIA._id, ADMIN_LOGIN);
+
+    cy.insertOneDeal(MOCK_DEAL_MIA, MOCK_MAKER_TFM)
+      .then((insertedDeal) => {
+        dealId = insertedDeal._id;
+
+        const { dealType, mockFacilities } = MOCK_DEAL_MIA;
+
+        cy.createFacilities(dealId, mockFacilities, MOCK_MAKER_TFM).then((createdFacilities) => {
+          dealFacilities = createdFacilities;
+        });
+
+        cy.submitDeal(dealId, dealType);
+      });
+  });
+
+  beforeEach(() => {
+    cy.login(MOCK_USERS[0]);
+    cy.visit(relative(`/case/${dealId}/deal`));
+  });
+
+  after(() => {
```

Comment:
do we need to delete any deals after the tests finish executing?

Reply:
I don't think so, it didn't in the original and doesn't affect any of the other running tests